frontend/redux/actions.js

```diff
@@ -57,6 +57,30 @@ export const requestAttributesFailed = (error, retryAction) => ({
   error: true,
 });
 
+/** @prop {string} Action to trigger a request for face data */
+export const REQUEST_OVERALL_STATS = 'REQUEST_OVERALL_STATS';
+export const requestOverallStats = () => ({
+  type: REQUEST_OVERALL_STATS,
+});
+
+/** @prop {string} Action to populate the annotations within the store when the request succeeds */
```

Comment:
These comments look like they were copied wholesale from other actions—they should be removed or updated to reflect what is being requested